app/Models/Competition.php

```diff
@@ -12,14 +11,14 @@ class Competition extends Model
      *
      * @var array
      */
-    protected $fillable = ['campaign_id', 'campaign_run_id', 'start_date', 'end_date'];
+    protected $fillable = ['contest_id', 'competition_start_date', 'competition_end_date'];
```

Comment:
Not a huge deal. I think it's fine. If we eventually decide against it, it's an easy fix as @deadlybutter pointed out. Plus it's not the actual contest id being assigned, so should be fine.